### PR TITLE
MNM documentation update for Birthday Week 2024! Let's Go!

### PR DESCRIPTION
**Pull request by cdraper-cloudflare**
### Documentation checklist

- [ ] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.
- [ ] If a larger change - such as adding a new page- an issue has been opened in relation to any incorrect or out of date information that this PR fixes.
- [ ] Files which have changed name or location have been allocated [redirects](https://developers.cloudflare.com/pages/configuration/redirects/#per-file).

### CODE REVIEW

**Comment by cdraper-cloudflare** [comment]
Hey Marcio! I got caught up in Q4 planning, and I forgot to file a PCX ticket for the free version of Magic Network Monitoring becoming generally available to everyone for Birthday Week. Sorry about that 😅.

I figured the best thing for me to do is to submit the PR directly to make your life easier given that this is very last minute. The announcement goes live at around 9:00 AM EST today. It would be great if the Developer Documentation could be updated either before or a couple of hours after the announcement. Let me know if that is possible!

There is one part of the PR that is a little bit bare bones. It's the new tutorial for customers to send encrypted network flow data to Cloudflare. I wanted to make sure this content went out with the Birthday Week update since the engineering team worked really hard to get this feature released before the LDW and just barely made it. I plan on going back later this week and filling in this tutorial with more detailed information.

Thanks again for your help! I really appreciate it. Let me know if you have any questions.

Cheers,
Chris Draper

File: src/content/docs/magic-network-monitoring/faq.mdx [hunk 1]
```diff
@@ -4,9 +4,21 @@ title: FAQ
...
 ---
 
+## Can I send NetFlow/sFlow data to Cloudflare in a secure, encrypted way?
+
+Yes. Both enterprise and free customers can send encrypted network flow data to Cloudflare.
+
+Enterprise customers with Magic Transit or Magic WAN are able to send encrypted network flow data via an IPsec tunnel to Cloudflare's network. You can achieve this by:
+
+1. Configuring your [NetFlow](/magic-network-monitoring/routers/netflow-ipfix-config/) or [sFlow](/magic-network-monitoring/routers/sflow-config/) data to be sent to Cloudflare’s network for parsing.
+2. Directing that network flow data to be sent over a [Magic Transit IPsec tunnels](/magic-transit/how-to/configure-tunnels/) or [Magic WAN IPsec tunnels](/magic-wan/configuration/manually/how-to/configure-tunnels/) to Cloudflare's network.
```
Comment (marciocloudflare) [inline src/content/docs/magic-network-monitoring/faq.mdx]:
```suggestion
2. Directing that network flow data to be sent over [Magic Transit IPsec tunnels](/magic-transit/how-to/configure-tunnels/) or [Magic WAN IPsec tunnels](/magic-wan/configuration/manually/how-to/configure-tunnels/) to Cloudflare's network.
```

File: src/content/docs/magic-network-monitoring/index.mdx [hunk 2]
```diff
@@ -6,24 +6,33 @@ sidebar:
...
+Improve your network visibility and detect DDoS attacks based on traffic flows.
 
 </Description>
 
 <Plan type="all" />
 
 Magic Network Monitoring provides visibility into your network traffic by analyzing network <GlossaryTooltip term="flow data">flow data</GlossaryTooltip> sent from a customer’s routers. Magic Network Monitoring supports NetFlow v5, NetFlow v9, IPFIX, and sFlow.
 
-Enterprise customers can use Magic Network Monitoring with [Magic Transit on-demand](/magic-transit/on-demand/) to monitor their network, identify volumetric DDoS attacks, and activate Magic Transit on-demand to mitigate those attacks.
+Magic Network Monitoring is generally available to everyone with a Cloudflare account by default. You can login to your Cloudflare dashboard, select your account, then go to **Analytics & Logs** > **Magic Monitoring** to get started.
```
Comment (marciocloudflare) [inline src/content/docs/magic-network-monitoring/index.mdx]:
```suggestion
Magic Network Monitoring is generally available to everyone with a Cloudflare account by default. You can log in to your Cloudflare dashboard, select your account, then go to **Analytics & Logs** > **Magic Monitoring** to get started.
```

File: src/content/docs/magic-network-monitoring/magic-network-monitoring-free.mdx [hunk 3]
```diff
@@ -3,18 +3,16 @@ title: Free version
...
+The free version of Magic Network Monitoring (MNM) is generally available to everyone with a Cloudflare account by default.
 
-If you would like your request for early access to MNM to be prioritized, please join [Cloudflare’s Discord server](https://discord.com/invite/cloudflaredev) and ask about early access in the MNM Discord channel. We prioritize Discord community members because Discord creates a great environment for engaging with users, helping with configuration, answering questions, and collecting valuable product feedback.
+All free customers are encouraged to join [Cloudflare’s Discord server](https://discord.com/invite/cloudflaredev) to discuss the use cases, configuration, and troubleshooting of Magic Network Monitoring. Free customers are always welcome to provide product feedback and discuss feature requests. The product and engineering team that works on Magic Network Monitoring regularly engages with the Discord community.
```
Comment (marciocloudflare) [inline src/content/docs/magic-network-monitoring/magic-network-monitoring-free.mdx]:
```suggestion
All free customers are encouraged to join [Cloudflare's Discord server](https://discord.com/invite/cloudflaredev) to discuss the use cases, configuration, and troubleshooting of Magic Network Monitoring. Free customers are always welcome to provide product feedback and discuss feature requests. The product and engineering team that works on Magic Network Monitoring regularly engages with the Discord community.
```

**Review by marciocloudflare — APPROVED** [review]

File: src/content/docs/magic-network-monitoring/index.mdx [hunk 4]
```diff
@@ -6,24 +6,33 @@ sidebar:
...
 
-Enterprise customers can use Magic Network Monitoring with [Magic Transit on-demand](/magic-transit/on-demand/) to monitor their network, identify volumetric DDoS attacks, and activate Magic Transit on-demand to mitigate those attacks.
+Magic Network Monitoring is generally available to everyone with a Cloudflare account by default. You can log in to your Cloudflare dashboard, select your account, then go to **Analytics & Logs** > **Magic Monitoring** to get started.
 
-Magic Network Monitoring is automatically enabled for all Magic Transit and Magic WAN enterprise customers. Any enterprise customers without Magic Transit or Magic WAN that are interested in testing Magic Network Monitoring can receive access to the Magic Network Monitoring [free version](/magic-network-monitoring/magic-network-monitoring-free/) by submitting a request to their Cloudflare account team.
+Everyone can use the [free version](/magic-network-monitoring/free-version) of Magic Network Monitoring in a home network, network lab, or business to get end to end visibility across their network traffic. Potential enterprise customers are encouraged to use the free version to run a proof of concept.
```
Comment (marciocloudflare) [inline src/content/docs/magic-network-monitoring/index.mdx]:
```suggestion
Everyone can use the [free version](/magic-network-monitoring/magic-network-monitoring-free/) of Magic Network Monitoring in a home network, network lab, or business to get end to end visibility across their network traffic. Potential enterprise customers are encouraged to use the free version to run a proof of concept.
```

**Review by marciocloudflare — APPROVED** [review]

**Pull request merged**